dracut/30ignition/ignition-files.service

```diff
@@ -5,30 +5,14 @@ DefaultDependencies=false
 # Ignition files stage starts after partitioning is completed.
 Requires=initrd-root-fs.target
 After=initrd-root-fs.target
+Before=initrd-parse-etc.service
+
+Requires=ignition-disks.service
+After=ignition-disks.service
 
 # Network may be required to fetch userdata content.
 After=network.target
 
-Requires=ignition-remount-sysroot.service
-After=ignition-remount-sysroot.service
-Requires=ignition-remount-sysroot-var.service
-After=ignition-remount-sysroot-var.service
-
-After=ostree-prepare-root.service
```

Comment:
I think this line is still relevant and should be kept somewhere else, not dropped.

In particular, I think that with this PR `ignition-disks` and `ostree-prepare-root` are now racing with each other. This may be a problem when ignition tries to format rootfs or /var.

I think a `Before=ostree-prepare-root.service` in ignition-disks would fix this, but please double check me.